syntax = "proto3";

package synapse;

enum DataType {
    kDataTypeUnknown = 0;
    kAny = 1;
    kBroadband = 2;
    kSpiketrain = 3;
    kTimestamps = 4;
    kImage = 5;
}